glfw/src/Joystick.zig

```diff
@@ -424,10 +418,7 @@ pub inline fn updateGamepadMappings(gamepad_mappings: [*:0]const u8) Error!void
 pub inline fn isGamepad(self: Joystick) bool {
     internal_debug.assertInitialized();
     const is_gamepad = c.glfwJoystickIsGamepad(@enumToInt(self.jid));
-    getError() catch |err| return switch (err) {
-        Error.InvalidEnum => unreachable, // intentionally invalid enum value
-        else => unreachable,
-    };
+    getError() catch unreachable;
```

Comment:
Thought for the future: we should change all such locations like this to a @panic maybe so we know what error occurred (e.g. if we see this in the wild, what error caused it? which one is not being handled?)

Reply:
That's a great idea; though I think that deserves its own PR, dedicated to just replacing all the 'catch unreachable's.